plugins/inputs/proxmox/proxmox.go

```diff
@@ -49,9 +52,10 @@ func (px *Proxmox) Gather(acc telegraf.Accumulator) error {
 }
 
 func (px *Proxmox) Init() error {
-
+	// Set hostname as default node name for backwards compatibility
 	if px.NodeName == "" {
-		return errors.New("node_name must be configured")
+		hostname, _ := os.Hostname()
+		px.NodeName = hostname
 	}
```

Comment:
Just a thought, if you move this to the init() function below, you don't need the `""` check, and it'll allow users to unset it if they don't want it